lib/handlers/router-entities/route-caching/dynamo-route-caching-provider.ts

```diff
@@ -230,6 +249,26 @@ export class DynamoRouteCachingProvider extends IRouteCachingProvider {
         },
       }
 
+      return { putParams, jsonCachedRoutes }
+    } else {
+      return null
+    }
+  }
+
+  /**
+   * Implementation of the abstract method defined in `IRouteCachingProvider`
+   * Attempts to insert the `CachedRoutes` object into cache, if the CachingStrategy returns the CachingParameters
+   *
+   * @param cachedRoutes
+   * @param amount
+   * @protected
+   */
+  protected async _setCachedRoute(cachedRoutes: CachedRoutes, amount: CurrencyAmount<Currency>): Promise<boolean> {
+    const cachedRouteDbEntry = this.generateCachedRouteDbEntry(cachedRoutes, amount)
+
+    if (cachedRouteDbEntry) {
+      const { putParams, jsonCachedRoutes } = cachedRouteDbEntry
+
       log.info(
         { putParams, cachedRoutes, jsonCachedRoutes },
```

Comment:
I would remove `jsonCachedRoutes` from this log to avoid the need of passing it down in the `generateCachedRouteDbEntry` function.